core/container/dockercontroller/dockercontroller.go

```diff
@@ -26,16 +26,32 @@ import (
 	"github.com/hyperledger/fabric/core/container/ccintf"
 	cutil "github.com/hyperledger/fabric/core/container/util"
 	"github.com/op/go-logging"
+	"github.com/spf13/viper"
 	"golang.org/x/net/context"
 )
 
-var dockerLogger = logging.MustGetLogger("dockercontroller")
+var (
+	dockerLogger = logging.MustGetLogger("dockercontroller")
+	hostConfig   = new(docker.HostConfig)
+)
 
 //DockerVM is a vm. It is identified by an image id
 type DockerVM struct {
 	id string
 }
 
+func init() {
+	err := viper.UnmarshalKey("vm.docker.hostConfig", hostConfig)
+	if err != nil {
+		dockerLogger.Fatalf("Load docker HostConfig wrong, error: %s", err.Error())
+	}
+
+	if hostConfig.NetworkMode == "" {
+		hostConfig.NetworkMode = "host"
+	}
+	dockerLogger.Debug("Load docker HostConfig: %+v", hostConfig)
+}
+
```

Comment:
Looks like Unmarshall and pass the docker.HostConfig directly to docker will allow users to add any of its 48 properties directly in core.yaml to control container process in a SUPPORTED fashion.  ("Privileged" is one such property.)

My suggestion would be to not unmarshal but use only the specific properties such as Host, Dns that are declared in the yaml to construct hostConfig.